### ISSUE TITLE
Unexpcted config result while apply multiple 'when'

### ISSUE THREAD

**Issue by tutufool** [lukas-krecan/JsonUnit#234]
code like:

```
    	Configuration config = Configuration.empty().when(path("aaa"), thenIgnore()).when(path("bbb"), thenIgnore());
    	
    	System.out.println(config.getPathsToBeIgnored());
```

output is:

`[bbb]`

while I expect it to be:

`[aaa, bbb]`

Is it a bug? looks like the latter one override the previous one?

**Comment by lukas-krecan** [comment]
It's a bug caused by unexpected usage pattern. There is `paths()` method for specifying multiple paths.

**Comment by lukas-krecan** [comment]
Released as 2.18.1
